react-native/Fable.Helpers.ReactNative.fs

```diff
@@ -157,722 +359,1335 @@ module Props =
+        | Source of U3<WebViewUriSource, WebViewHtmlSource, float>
+        | MediaPlaybackRequiresUserAction of bool
+        | ScalesPageToFit of bool
+        | Ref of Ref<obj>
+        interface IWebViewProperties
+
+    [<KeyValueList>]
+    type SegmentedControlIOSProperties =
+        | Enabled of bool
+        | Momentary of bool
+        | OnChange of Func<NativeSyntheticEvent<NativeSegmentedControlIOSChangeEvent>, unit>
+        | OnValueChange of Func<string, unit>
+        | SelectedIndex of float
         | TintColor of string
-        | Opacity of float
-        interface IImageStyle
+        | Values of ResizeArray<string>
+        | Ref of Ref<SegmentedControlIOS>
+        interface ISegmentedControlIOSProperties
 
     [<KeyValueList>]
-    type ITextStyle =
-        interface end
+    type NavigatorIOSProperties =
+        | BarTintColor of string
+        | InitialRoute of Route
+        | ItemWrapperStyle of ViewStyle list
+        | NavigationBarHidden of bool
+        | ShadowHidden of bool
+        | TintColor of string
+        | TitleTextColor of string
+        | Translucent of bool
+        | Style of IStyle list
+
+    module ActivityIndicator =
+        [<KeyValueList>]
+        type ActivityIndicatorProperties =
+            | Animating of bool
+            | Color of string
+            | HidesWhenStopped of bool
+            | Size of (* TODO StringEnum small | large *) string
+            | Style of IStyle list
+            | Ref of Ref<ActivityIndicator>
+            interface IViewProperties
+
+
+        [<KeyValueList>]
+        type ActivityIndicatorIOSProperties =
+            | Animating of bool
+            | Color of string
+            | HidesWhenStopped of bool
+            | OnLayout of Func<obj, unit>
+            | Size of (* TODO StringEnum small | large *) string
+            | Style of IStyle list
+            | Ref of Ref<ActivityIndicatorIOS>
+            interface IViewProperties
 
     [<KeyValueList>]
-    type TextStyle =
-        // ViewStyle
-        | AlignItems of Alignment
-        | AlignSelf of Alignment
-        | BorderBottomWidth of float
-        | BorderLeftWidth of float
-        | BorderRightWidth of float
-        | BorderTopWidth of float
-        | BorderWidth of float
-        | Bottom of float
-        | Flex of int
-        | FlexDirection of string
-        | FlexWrap of string
+    type DatePickerIOSProperties =
+        | Date of DateTime
+        | MaximumDate of DateTime
+        | MinimumDate of DateTime
+        | MinuteInterval of float
+        | Mode of (* TODO StringEnum date | time | datetime *) string
+        | OnDateChange of Func<DateTime, unit>
+        | TimeZoneOffsetInMinutes of float
+        | Ref of Ref<DatePickerIOS>
+        interface IDatePickerIOSProperties
+
+    [<KeyValueList>]
+    type DrawerLayoutAndroidProperties =
+        | DrawerBackgroundColor of string
+        | DrawerLockMode of DrawerLockMode 
+        | DrawerPosition of obj
```

Comment:
Sorry, the TS definitions I pulled didn't have some of those and somehow I missed it reconciling the differences. Amended the commit.